### ISSUE TITLE
"Defiler's Talisman" lvl 28/38/48 variants missing from "Rutherford Twing" in 1.7

### ISSUE THREAD

**Issue by niwolos** [elysium-project/server#3185]
# TL;DR:
- [**_Rutherford_Twing_**](http://wowwiki.wikia.com/wiki/Rutherford_Twing) || [(cache March-16th-2005)](https://web.archive.org/web/20060316172134/http://wow.allakhazam.com:80/db/mob.html?wmob=15126)
**is missing:**
- [**_Defiler's Talisman_ (level 28)**](http://wowwiki.wikia.com/wiki/Defiler%27s_Talisman_(level_28)) || [c _72s30c_](https://web.archive.org/web/20060316172134/http://wow.allakhazam.com:80/db/mob.html?wmob=15126)
- [**_Defiler's Talisman_ (level 38)**](http://wowwiki.wikia.com/wiki/Defiler%27s_Talisman_(level_38)) || [1g52s30c ](https://web.archive.org/web/20060316172134/http://wow.allakhazam.com:80/db/mob.html?wmob=15126)
- [**_Defiler's Talisman_ (level 48)**](http://wowwiki.wikia.com/wiki/Defiler%27s_Talisman_(level_48)) || [2g82s30c](https://web.archive.org/web/20060316172134/http://wow.allakhazam.com:80/db/mob.html?wmob=15126)
for sale

# Description

On Elysium (recently patched to 1.7) ["**Rutherford Twing**"](http://wowwiki.wikia.com/wiki/Rutherford_Twing) aka the Horde AB rep vendor in Arathi Highlands 
is missing: the [28](http://wowwiki.wikia.com/wiki/Defiler%27s_Talisman_(level_28))/[38](http://wowwiki.wikia.com/wiki/Defiler%27s_Talisman_(level_38))/[48](http://wowwiki.wikia.com/wiki/Defiler%27s_Talisman_(level_48)) versions of: the ["**Defiler's Talisman**"](http://db.vanillagaming.org/?item=21120) phys-absorb trinket.

He currently only possesses the [28](http://wowwiki.wikia.com/wiki/Defiler%27s_Chain_Girdle_(level_28))/[cloth lvl 38](http://wowwiki.wikia.com/wiki/Defiler%27s_Cloth_Girdle_(level_38))([mail lvl 40](http://wowwiki.wikia.com/wiki/Defiler%27s_Chain_Girdle_(level_40)))/[48](http://wowwiki.wikia.com/wiki/Defiler%27s_Chain_Greaves_(level_48)) variants of the Belt & Boots (Honored&Revered rewards)
which are stated on the wikis below to have been added in the same patch at the same time as the trinket.

### wiki articles on the release date / patch
**http://wowwiki.wikia.com/wiki/Defiler%27s_Talisman_(level_28)**
**https://wow.gamepedia.com/Defiler%27s_Talisman_(level_28)**

### old versions / web cache

**https://web.archive.org/web/20060316172134/http://wow.allakhazam.com:80/db/mob.html?wmob=15126**

The earliest available cache of the vendor itself on allakhazam revealed the lvl 28/38/48 variants of the "Defiler's Talisman" for sale - priced at 72s30c / 1g52s30c / 2g82s30c.

The earliest cache done on **March 16th 2006** => after patch 1.9, before patch 1.10.

### [release dates:](http://wowwiki.wikia.com/wiki/Patches/1.x)
[1.7](http://wowwiki.wikia.com/wiki/Patch_1.7.0) - matches September 13th on [wowwiki](http://wowwiki.wikia.com/wiki/Defiler%27s_Talisman_(level_28))
[1.7.1](http://wowwiki.wikia.com/wiki/Patch_1.7.1) - matches September 22nd on [gamepedia](https://wow.gamepedia.com/Defiler%27s_Talisman_(level_28))

Therefore you may conclude that - based on the mentioned release dates, and the fact that the trinket existed as early as 1.9 (based on earlierst ever snapshots of the vendor), along with the fact that lvl 28/38/48 variants of the belt and boots were supposedly added with the same patch as the fitting belt/boot:
The missing lvl 28/38/48 variants of the **Defiler's Talisman** need to be added to **Rutherford Twing** as sold friendly-reputation rewards.

Thank you for reading.

**Comment by QAkvastskaft** [comment]
Lower level rewards for Arathi Basin were added in 1.9:
http://wowwiki.wikia.com/wiki/Patch_1.9.0
